tests/basic/pct.rs

```diff
@@ -94,3 +94,154 @@ fn yield_spin_loop_fair() {
 fn yield_spin_loop_unfair() {
     yield_spin_loop(false);
 }
+
+#[test]
+#[should_panic(expected = "called `Option::unwrap()` on a `None`")]
+// Based on Fig 1(a) of the PCT paper.  We model NULL pointer dereference with an Option unwrap
```

Comment:
Maybe use `expect` so we can target the exact "null dereference" in `should_panic` rather than any unwrap failing (same for the other tests)

Reply:
Done.